test/system/src/test/java/io/pravega/test/system/PravegaTest.java

```diff
@@ -72,55 +69,42 @@
      */
     @Environment
     public static void initialize() {
+        log.info("Environment: Invoking tests with security enabled: {}", Utils.SECURITY_ENABLED);
```

Comment:
Did you mean to say something like "Setting up environment with auth enabled"? The existing log message can be misleading if the environment creation fails (since it says "Invoking tests..."). Also, why are you adding the "Environment" prefix to the log statement? I think it is unnecessary as method names can be automatically logged via an appropriate log format configuration via appender encoder pattern in the logging configuration file.

Reply:
Added this log to make sure system property (securityEnabled) set by gradle when starting the tests, is received correctly inside initialize().
Nevertheless, the test method does not automatically receive this value unless it is set on the test pod at the time of creation... see changes to class K8SequentialExecutor

Reply:
Sure. Having a log statement here makes sense. The issue I have is the message in the log entry, which is misleading (At this point, it is not that the tests are being invoked.)

Reply:
removed log